### PR TITLE
Completion of Logs Page containing Logs.tsx and Sidebar.tsx, Currently using mock data.

### PR DESCRIPTION
**Pull request by davidchuang5**
… css file to logsfile folder

### CODE REVIEW

File: ui/src/pages/Logfiles/Logs.tsx [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+      value: 37,
+      label: '2:00am',
+    },
+    {
+      value: 100,
+      label: '3:00am',
+    },
+  ];
+
+  function valuetext(value: number) {
```
Comment (scotthallock) [inline ui/src/pages/Logfiles/Logs.tsx:204]: What is the purpose of this function?

**Review by scotthallock — CHANGES_REQUESTED** [review]

**Comment by scotthallock** [comment]
For future PRs it would be nice if you added a screenshot of the UI to the PR description, that way we can easily see what was added.

File: ui/src/pages/Logfiles/Logs.tsx [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+    return `${value}`;
+  }
+
+  return (
+    <>
+      <div className="parentDiv">
```
[the remark this answers is not in the document]
Comment (davidchuang5) [reply ui/src/pages/Logfiles/Logs.tsx:210]: Still using the custom CSS only for the sake of positioning the drawer to the left of the logs display

File: ui/src/pages/Logfiles/Logs.tsx (hunk 1 above)
Comment (davidchuang5) [reply ui/src/pages/Logfiles/Logs.tsx:204]: Displays the time on the slider

**Comment by davidchuang5** [comment]
Screenshot 2023-06-14 at 11 16 55 AM
Screenshot 2023-06-14 at 11 16 38 AM

**Review by leejun07 — APPROVED** [review]

**Comment by davidchuang5** [comment]
Screenshot 2023-06-15 at 7 56 26 PM

**Comment by davidchuang5** [comment]
Screenshot 2023-06-15 at 7 56 53 PM

**Comment by davidchuang5** [comment]
Screenshot 2023-06-16 at 10 20 08 AM
Screenshot 2023-06-16 at 10 20 25 AM

**Comment by scotthallock** [comment]
@davidchuang5 an earlier PR you made added a `package-lock.json` file to the root directory, but the root direction should not have a `package.json` or a `package-lock.json`. (Only the `/ui` and `/backend` folders need package.json because they are npm projects).

Can you remove these files from the root directory?

**Comment by davidchuang5** [comment]
Screenshot 2023-06-16 at 10 28 22 AM

**Review by scotthallock — APPROVED** [review]
LGTM

**Pull request merged**